packages/web_benchmarks/lib/src/metrics.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The metrics measured for each benchmark.
```

Comment:
Ooh, `enum` is so much better! Thanks for refactoring this.

Bonus points: change the style of how these metric names are described (including the top-level enum dartdoc). We claim that these are "metrics", but they are not. They are "labels" or "names" for the various metrics we collect.